Addon/MultiColumnContentAreaRenderer.cs

```diff
@@ -62,16 +62,19 @@ public void RenderContentAreaItemsInternal(IHtmlHelper htmlHelper, IEnumerable<C
             IEnumerable<ContentAreaItem> contentAreaItems)
         {
             IRenderingContentAreaContext currentContext = DefaultContentAreaContext.Instance;
-
             foreach (var current in contentAreaItems)
             {
                 var content = _contentAreaLoader.Get(current);
                 if (content is IRenderingLayoutBlock asLayoutBlock)
                 {
                     var newContext = asLayoutBlock.NewContext();
+                    htmlHelper.ViewContext.ViewData[RenderingMetadataKeys.Layout.ContentLink] = current.ContentLink;
```

Comment:
Would it be possible to put all metadata under a single key?

Reply:
I was able to move the metadata into less keys. 
But since we need to "parent" context of the layout we still need two keys in the viewdata..